### ISSUE TITLE
cannot use more than one word in value

### ISSUE THREAD

**Issue by born2ngopi** [cacing/cacing#2]
cannot use more than one value

![issue](https://user-images.githubusercontent.com/38811722/106381980-ff359500-63ee-11eb-985b-54323d230ced.png)

**Comment by needkopi** [comment]
same problem if we use expired, example :
``` bash
SET test this is value 6s
```

maybe we must use quote or double quote to check that is value or not, example
``` bash
SET test "this is value" 6s
```

**Comment by hadihammurabi** [comment]
Amazing. I don't think about this before.

In simple way, we can get the last word as expire time. But, this is not a good solution. Then, we need to create more logics to parse the command.

Any tough?

**Comment by needkopi** [comment]
maybe we can use regexp like [this](https://play.golang.org/p/5SohCAYMUGn)

**Comment by hadihammurabi** [comment]
yes, agree, we can do it at client side and send to server with json formatted message.
then we need to re-format the message payload for execution.

for example:
```json
{
  "t": "SIGNAL_EXEC",
  "u": "uuid-saasd87d8a7s8d78as-adaus8a0sa8d",
  "p": {
    "exec": "SET",
    "args": [ "user1", "someones fullname", "6h"],
  },
}
```

it think it will comfortable to handle the message at server side.

how? any suggestions?

**Comment by needkopi** [comment]
yeah, i think it's good if **p** is slice

**Comment by hadihammurabi** [comment]
alright, we clear here. let's see when it will happen.

**Comment by hadihammurabi** [comment]
I made it Redis compatible (49da035e2a3f5d86303121ea926eccbcc2c86a54) and guess what happen?
Yeah, this issue solved.

<img width="818" height="331" alt="Image" src="https://github.com/user-attachments/assets/068fd9ec-ebe4-4628-9e21-4a607ed29d71" />
